thunder/core/update_aliases.py

```diff
@@ -39,6 +39,7 @@ def _get_new_aliases(aliases, trace):
 
 
 def _is_inplace_op(bsym):
+    # TODO: Handle higher order bsyms containing inplace ops
```

Comment:
would this mean, the current implementation isn't capable of handling bsyms any of whose sub-bsyms are in-place?

Reply:
Yes. `batch_norm` is one example (see the new [xfailed test](https://github.com/Lightning-AI/lightning-thunder/pull/2777#discussion_r2571794068)).